src/components/dashboard/FeedList.js

```diff
@@ -11,9 +11,8 @@ const VIEWABILITY_CONFIG = {
   viewAreaCoveragePercentThreshold: 100,
   waitForInteraction: true,
 }
-
+const { width } = Dimensions.get('window')
```

Comment:
Dimensions.get from window of from screen maybe is better? I'm thinking on desktop screens, but not sure which is the best one, it's just a question

Reply:
Not sure what you mean. I think in this case it's used to calculate something related to the window's width (which could or could not be the same as the screen's).